tests/helpers/__init__.py

```diff
@@ -41,15 +48,109 @@ def create_random_packet(root, name="data", *, parameters=None, packet_id=None):
     return p.id
 
 
+## Create a chain of packets a, b, c, ... that depend on each other
+def create_random_packet_chain(root, length, base=None):
```

Comment:
This could be a bit more concise and less noisy using the `create_packet` function above. You can also simplify the `id_use` code by just using the previous iteration's ID. Also just let the ID be chosen randomly by the `Packet` class.

```python
for i in range(length):
  with create_packet(root, name) as p:
     # Write some data
     if base is not None:
       packet.use_dependency(base, {"input.txt": "data.txt"})
  ids[name] = p.id
  base = p.id
return ids
```

Reply:
Thanks, that's way better! I've used this and just written the data directly instead of writing a script as suggested above. Looking way better now